conf/test.config

```diff
@@ -43,4 +43,5 @@ params {
     skip_checkm = true
     skip_raw_read_metrics = false
     skip_polishing = false
+    skip_allele_calling = true // No allele database or sample data specified yet
```

Comment:
Why is allele calling skipped if the locidex allele database is set?

Reply:
good question! I am enabling this now

Reply:
fixed in: [215a70b](https://github.com/phac-nml/mikrokondo/pull/96/commits/215a70b19612fe50ae48c5194cad0658665486b6)